src/main/deploy/auton/auton.dtd

```diff
@@ -18,7 +18,7 @@
           heading           CDATA "0.0"
           pathname          CDATA #IMPLIED
           alignmentMethod   ( ROTATE | STRAFE ) "ROTATE"
-          pipeline          ( UNKNOWN | OFF | APRIL_TAG | CONE_NODE | CONE | CUBE ) "APRIL_TAG"
+          visionAligment          ( UNKNOWN | NOTE | SPEAKER ) "APRIL_TAG"
```

Comment:
Default value ("APRIL_TAG") should be one of the options.  In this case "UNKNOWN" would be appropriate.